apps/src/aichat/views/InfoTooltipIcon.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import FontAwesomeV6Icon from '@cdo/apps/componentLibrary/fontAwesomeV6Icon/FontAwesomeV6Icon';
+import {TooltipProps, WithTooltip} from '@cdo/apps/componentLibrary/tooltip';
+
+import styles from './info-tooltip-icon.module.scss';
+
+interface InfoTooltipIconProps {
+  id: string;
+  tooltipText: string;
+  direction?: TooltipProps['direction'];
```

Comment:
Does this technically only support `onRight` currently?

Reply:
Well so currently only `onRight` will receive custom styling (`text-align: left`) but otherwise the direction will still determine where the tooltip appears.

Reply:
Ah got it!